.gitignore

```diff
@@ -5,3 +5,4 @@ config/prod.php
 dev_db.sqlite3
 build/
 .sass-cache
+assets/vid/IIB 10-06-16 Segment 5A.mp4
```

Comment:
What's going on here? If this video doesn't need to exist in your source tree, it shouldn't be in the repo. Did you at one point try to upload the video before deciding to use vimeo?